app.js

```diff
@@ -1,166 +0,0 @@
-    const reply_input = document.querySelector("#reply-box");
-    reply_input.addEventListener("keypress", (e) => {
-      if (checkEnterKey(e)) {
-        let message = reply_input.value;
-        if (message != "") {
-          let comment_obj = {
-            name: "Ankit",
-            src: "http://facebook.com/",
-            message: message,
-          };
-          addReply(comment_obj, index); //adding reply in local stroge
-          document.querySelector("#reply-box").value = "";
-          nestedCommentForm.style.display = "none";
-        }
-      }
-    });
-  };
-
-  //render all comments on UI
-  let renderComments = () => {
-    let comment_lists = getItemFromLocalStorage();
-    let comment_list = comment_lists
-      .map((comment, index) => {
-        return `<li class="comment_list">
-        <div class="parent-comment">
-            <div class="user_avtar">
-                <img
-                    src="https://instagram.fdel18-1.fna.fbcdn.net/v/t51.2885-15/e35/95168755_1188313104848732_7163586086101228730_n.jpg?_nc_ht=instagram.fdel18-1.fna.fbcdn.net&_nc_cat=108&_nc_ohc=E04Bpx3IeJYAX8m_BP-&oh=3486a37e1f6b0807382ed237660dc258&oe=5EDC7BFF">
-            </div>
-            <div class="user-comments">
-                <h3>Amir Sohel</h3>
-                <p>${comment.message}</p>
-                <a onclick="reply(${index})" class="reply">Reply</a>
-                <a onclick="deleteComment(${index})" class="delete">Delete</a>
-            </div>
-        </div>
-        ${
-          comment && comment.children
-            ? comment.children
-                .map(
-                  (children, childIndex) => `<div class="sub-comment">
-        <div class="user_avtar">
-            <img
-                src="https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fstatic.onecms.io%2Fwp-content%2Fuploads%2Fsites%2F6%2F2018%2F04%2Fnup_181708_1056-2000.jpg">
-        </div>
-        <div class="user-comments">
-            <h3>Johny Smith</h3>
-            <p>${children.message}</p>
-            <a onclick="reply(${index})" class="reply">Reply</a>
-            <a onclick="deleteChild(${childIndex})" class="delete">Delete</a>
-        </div>
-    </div>`
-                )
-                .join("")
-            : ""
-        }
-    </li>`;
-      })
-      .join("");
-
-    //Bind wiht html
-    comment_wrapper.innerHTML = comment_list;
-    //get list from bottom
-    comment_wrapper.scrollTo({
-      top: comment_wrapper.scrollHeight,
-      behavior: "smooth",
-    });
-  };
-
-  //adding new comment on press enter
-  comment_box.addEventListener("keypress", (e) => {
-    if (checkEnterKey(e)) {
-      let message = comment_box.value;
-      let comment_obj = {
-        name: "Amir Sohel",
-        src: "http://google.com/",
-        message: message,
-      };
-      comment_box.value = "";
-      addComment(comment_obj);
-    }
-  });
-
-  //Adding functions in global scope
```

Comment:
move this to the top and instead of assigning it to the window, create an object like globalFuncs